AutoReplayUploader/AutoReplayUploaderPlugin.cpp

```diff
@@ -326,18 +522,90 @@ void ReplayFileUploadData::OnRequestComplete(HTTPRequestCompleted_t * pCallback,
 		}
 		requester->gameWrapper->Toast("Autoreplayuploader", message, logo_to_use, 3.5f, toastType);
 	}
+#endif
 }
 
-void AuthKeyCheckUploadData::OnRequestComplete(HTTPRequestCompleted_t * pCallback, bool failure)
+#pragma region Utility Functions
+
+string GetPlaylistName(int playlistId) {
```

Comment:
Is there some thought to having a common library available of functions like these in the Bakkes mod API?

Reply:
Yep, this is definitely a feature request as other plugins such as the PluginManager are also gonna need it.  https://github.com/bakkesmodorg/PluginManager